### DISCUSSION TITLE
cartera firefly

### DISCUSSION THREAD

**Discussion by Josewliber** [iotaledger/firefly#7788] [Help & Support]
En la cartera de firefly, no puedo enviar los iota a otra cartera porque me dice que no se encuentra los nodos. ( que no se lo que es). y en la versión actualizada  firefly shimmer, la dirección no sale en iota, si no en smr.
 Con lo cual no puedo disponer de mis activos. ¿Sabe alguien la solución?

**Comment by begonaalvarezd** [comment]
Parece que estas usando una version antigua de Firefly, por favor actualiza a la ultima version compatible con el nuevo protocolo, v2.0.4

**Reply by Josewliber** [reply]
Buenas noches. 
si era eso en lo que me estaba confundiendo, ya lo he actualizado y todo bien.
Gracias
